### ISSUE TITLE
Fluorine - I see error/traceback when running minion in debug mode 

### ISSUE THREAD

**Issue by frogunder** [saltstack/salt#49081]
In Fluorine, before the key is accepted when running the minion in debug mode (also see it in the minion log), I see this error/traceback.
Key is accepted fine and I can run commands fine so its working correctly, just see/get this error/traceback
```
[ERROR   ] Exception in callback <functools.partial object at 0x23e97e0>
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/tornado/ioloop.py", line 591, in _run_callback
    ret = callback()
  File "/usr/lib64/python2.7/site-packages/tornado/stack_context.py", line 274, in null_wrapper
    return fn(*args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/tornado/ioloop.py", line 597, in <lambda>
    self.add_future(ret, lambda f: f.result())
  File "/usr/lib64/python2.7/site-packages/tornado/concurrent.py", line 214, in result
    raise_exc_info(self._exc_info)
  File "/usr/lib64/python2.7/site-packages/tornado/gen.py", line 215, in wrapper
    result = func(*args, **kwargs)
TypeError: run() got an unexpected keyword argument 'async'
[ERROR   ] Exception in callback <functools.partial object at 0x23e9890>
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/tornado/ioloop.py", line 591, in _run_callback
    ret = callback()
  File "/usr/lib64/python2.7/site-packages/tornado/stack_context.py", line 274, in null_wrapper
    return fn(*args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/tornado/ioloop.py", line 597, in <lambda>
    self.add_future(ret, lambda f: f.result())
  File "/usr/lib64/python2.7/site-packages/tornado/concurrent.py", line 214, in result
    raise_exc_info(self._exc_info)
  File "/usr/lib64/python2.7/site-packages/tornado/gen.py", line 215, in wrapper
    result = func(*args, **kwargs)
TypeError: run() got an unexpected keyword argument 'async'
```

### Setup
Install Fluorine. I used this commit: 1f668c0011394f508c3a5dc8062ca246f1dce0c6
```
wget -O bootstrap.sh https://bootstrap.saltstack.com
sh bootstrap.sh -g https://github.com/saltstack/salt.git -M -P git 1f668c0011394f508c3a5dc8062ca246f1dce0c6
```

### Steps to Reproduce Issue
Install salt, change minion config file and restart minion. If you run in debug mode you see it, if not check the minion log file

### Versions Report
```
Salt Version:
           Salt: 2018.11.0-257-g1f668c0
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: Not Installed
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.7.2
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: 0.28.2
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.5.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.5 (default, Aug  4 2017, 00:39:18)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.3.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4
 
System Versions:
           dist: centos 7.4.1708 Core
         locale: UTF-8
        machine: x86_64
        release: 3.10.0-693.el7.x86_64
         system: Linux
        version: CentOS Linux 7.4.1708 Core
```

**Comment by rallytime** [comment]
@gtmanfred Can you take a look at this one since you've been involved in this lately?

**Comment by gtmanfred** [comment]
This will be fixed by #49085
